lib/PuppeteerSharp/LaunchOptions.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace PuppeteerSharp
+{
+    public class LaunchOptions
+    {
+        public bool AppMode { get; set; }
+
+        public bool IgnoreHTTPSErrors { get; set; }
+
+        public bool IgnoreDefaultArgs { get; set; }
+
+        public bool Headless { get; set; } = true;
+
+        public string ExecutablePath { get; set; }
+
+        public int SlowMo { get; set; }
+
+        public string[] Args { get; set; } = Array.Empty<string>();
+
+        public bool HandleSIGINT { get; set; } = true;
```

Comment:
Let's add only properties we're using right now in the code. There are many guys here which are not being used.